hermes_core/tests/test_util_cdf.py

```diff
@@ -0,0 +1,156 @@
+"""Tests for cdf.py"""
+
+from pathlib import Path
+import pytest
+import numpy as np
+from spacepy.pycdf import CDF, Var, gAttrList, zAttrList
```

Comment:
Remember to add spacepy to the list of dependencies in the `pyproject.toml` file.